packages/suite/src/components/suite/Preloader/Preloader.tsx

```diff
@@ -72,7 +79,8 @@ export const Preloader = ({ children }: PreloaderProps) => {
     }
 
     if (
-        router.route?.app !== 'settings' &&
+        (router.route?.app === undefined ||
```

Comment:
Nit:

```suggestion
        (router.route?.app !== undefined &&
```

Isn't it more intuitive like this?